eng/configure-toolset.ps1

```diff
@@ -0,0 +1,24 @@
+. $PSScriptRoot\common\pipeline-logging-functions.ps1
```

Comment:
Starting to suspect you copied a number of eng/ files from dotnet/arcade even though most are specific to the slightly odd setup they have there. Might want to look at a simpler repo like dotnet/razor-compiler i.e. this repo probably needs Publishing.props, Version.Details.xml, Versions.props, and not much (if anything) more.